cards/card.html

```diff
@@ -0,0 +1,119 @@
+                    <img class="card-img-top" src="/images/swami.jpg" alt="">
+                    <div class="card-body">
+                        <p class="card-title">Youth of India</p>
+                        <p class="card-text">Rs 499 <span class="discount-para">(50% off)</span></p>
+                        <a href="#" class="btn-card">Add to cart</a>
+                    </div>
+                </div>
+            </div>
+
+            <div class="div-centered mrg-med">
+                <iframe
+                src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=seti&wt=none&l=auto&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522div-centered%2522%253E%250A%2520%2520%253Cdiv%2520class%253D%2522card-container%2522%253E%250A%2520%2520%2520%2520%253Cimg%2520class%253D%2522card-img-top%2522%2520src%253D%2522%252Fimages%252Fjs.jpg%2522%2520alt%253D%2522%2522%253E%250A%2520%2520%2520%2520%2520%253Cdiv%2520class%253D%2522card-body%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%253Cp%2520class%253D%2522card-title%2522%253EJavascript%253C%252Fp%253E%250A%2520%2520%2520%2520%2520%2520%2520%253Cp%2520class%253D%2522card-text%2522%253ERs%2520299%2520%253Cspan%2520class%253D%2522discount-para%2522%253E%2850%2525%2520off%29%253C%252Fspan%253E%253C%252Fp%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Ca%2520href%253D%2522%2523%2522%2520class%253D%2522btn-card%2522%253EAdd%2520to%2520cart%253C%252Fa%253E%250A%2520%2520%2520%2520%2520%2520%2520%253C%252Fdiv%253E%250A%2520%253C%252Fdiv%253E%250A%253Cdiv%2520class%253D%2522card-container%2522%253E%250A%2520%2520%253Cimg%2520class%253D%2522card-img-top%2522%2520src%253D%2522%252Fimages%252Fswami.jpg%2522%2520alt%253D%2522%2522%253E%250A%2520%2520%2520%253Cdiv%2520class%253D%2522card-body%2522%253E%250A%2520%2520%2520%2520%253Cp%2520class%253D%2522card-title%2522%253EYouth%2520of%2520India%253C%252Fp%253E%250A%2520%2520%2520%2520%253Cp%2520class%253D%2522card-text%2522%253ERs%2520499%2520%253Cspan%2520class%253D%2522discount-para%2522%253E%2850%2525%2520off%29%253C%252Fspan%253E%253C%252Fp%253E%250A%2520%2520%2520%2520%2520%253Ca%2520href%253D%2522%2523%2522%2520class%253D%2522btn-card%2522%253EAdd%2520to%2520cart%253C%252Fa%253E%250A%2520%2520%2520%2520%253C%252Fdiv%253E%250A%253C%252Fdiv%253E"
+                style="width: 850px; height: 502px; border:0; transform: scale(1); overflow:hidden;"
+                sandbox="allow-scripts allow-same-origin">
+              </iframe>
+            </div>
+
+            <div class="div-centered fs-sm">
+                <h2>Horizontal Card</h2>
+            </div>
+
+            <div class="div-centered">
+                <div class="vertical-card-container">
+                    <img class="card-img-top" src="/images/js.jpg" alt="">
+                    <div class="card-body">
```

Comment:
Here class should be vertical-card-body, right?. there is no margin for body container.